docs/guides/display-alerts.md

```diff
@@ -0,0 +1,87 @@
+---
+title: How to display Alerts in your App
+sidebar_label: How to display Alerts
+---
+
+To display an alert in your web application, you will need to use the Alert Component from the `@dhis2/ui` library. However, you do not need to install this library as the best practice is to display the alert using the `useAlert` hook from the `@dhis2/app-runtime` library.
+
+In this guide we'll walk you through, step by step, how to display an alert in your application and handle the different alerts states.
+
+## Prerequisites
+
+To be able to implement the alert in your application, you need to have the following:
+
+- A web application built using the DHIS2 Application Platform
+- A basic understanding of React hooks
+  
+If you do not yet have a web application built using the DHIS2 Application Platform, you can follow the [Getting Started](/docs/quickstart/quickstart-web) guide to create a new application.
+
+## Step 1: Import the `useAlert` hook
+
+The `useAlert` hook is a custom hook provided by the `@dhis2/app-runtime` library. To use it, you need to import it in your file.
+
+```js
+import { useAlert } from '@dhis2/app-runtime'
+```
+
+Now that you have imported the `useAlert` hook, you can use it in your component.
+
+## Step 2: Send a default alert notification
+
+Now that you have imported the `useAlert` hook, you can use it to send a default alert notification. The `useAlert` hook returns an object with two functions: `show` and `hide`.
+
+The `show` function is used to display the alert, while the `hide` function is used to hide the alert. After hiding (automatically or by code), the alert will be removed from the alert stack. But you can re-add it by calling the `show` function again. 
+
+```js
+const { show, hide } = useAlert('This is a default alert message')
```

Comment:
It would be good to indicate the context around this line - it needs to be added to a React functional component.  I think it might be more clear to also show the entire component in the `App.js` file, for example, so that it's clear what's being added.  Maybe this could show the default template App.js with a diff showing that this line was being added?